driver/driver_test.go

```diff
@@ -37,221 +52,504 @@ func TestDriver(t *testing.T) {
 var _ = BeforeSuite(func() {
 	err := common.HardResetHNS()
 	Expect(err).ToNot(HaveOccurred())
-})
-
-var _ = AfterSuite(func() {
-	err := common.HardResetHNS()
+	err = common.RestartDocker()
```

Comment:
I think this is in some commit in the middle? I think this is fixed in next commits